jobs/JGLOBAL_MARINE_BMAT

```diff
@@ -23,6 +23,14 @@ gcyc=${GDATE:8:2}
 export GDUMP="gdas"
 export GDUMP_ENS="enkf${GDUMP}"
 
+export NMEM_ENS_MAX=${NMEM_ENS}
+if [[ "${RUN}" == "gfs" ]]; then
+   NMEM_ENS=${NMEM_ENS_GFS}
+   export mem_offset=$((NMEM_ENS_GFS_OFFSET * cyc/6))
+else
+   export mem_offset=0
```

Comment:
Where is `mem_offset` used in the `marine_bmat` job flow?

Reply:
It's used in the background staging yaml for member rotation